### PR TITLE
ID-0000: Fix minor thing and styling in TemperatureChart

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: src/TemperatureChart/TemperatureChart.js [hunk 1]
```diff
@@ -66,12 +70,13 @@ const rangeLineStyle = {
             3,
             2,
         ],
+        strokeWidth: 1.5,
     },
     parent: {
-        border: '1px solid #ccc',
+        border: '1px solid #00000066',
```
Comment (dimaPortyanka) [inline src/TemperatureChart/TemperatureChart.js:76]: i see this color repits several times maybe we should move it to constant?

**Review by dimaPortyanka — DISMISSED** [review]

File: src/TemperatureChart/TemperatureChart.js (hunk 1 above)
Comment (nuria-balaguer) [reply src/TemperatureChart/TemperatureChart.js]: fixed

**Review by taraspaliy2020 — DISMISSED** [review]

File: src/TemperatureChart/__tests__/TemperatureChart.spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                    3,
+                    1,
+                ]}
+                excursion={4}
+                upperTempBound={8}
+                lowerTempBound={2}
+            />,
+        )
+
+        expect(wrapper).toMatchSnapshot()
```
Comment (dimaPortyanka) [inline src/TemperatureChart/__tests__/TemperatureChart.spec.js:65]: dont do two snapshot instead you can try to use find and identify explicitly what shouldn't or should be in markup

Comment (nuria-balaguer) [reply src/TemperatureChart/__tests__/TemperatureChart.spec.js]: fixed

**Review by taraspaliy2020 — DISMISSED** [review]

File: src/TemperatureChart/__tests__/TemperatureChart.spec.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    })
+
+    it('Should not render energyLevel if energyLevel undefined', () => {
+        const wrapper = mount(
```
Comment (dimaPortyanka) [inline src/TemperatureChart/__tests__/TemperatureChart.spec.js:48]: why mount is needed? isn't shallow enough?

**Review by taraspaliy2020 — APPROVED** [review]

File: src/TemperatureChart/__tests__/TemperatureChart.spec.js (hunk 3 above)
Comment (nuria-balaguer) [reply src/TemperatureChart/__tests__/TemperatureChart.spec.js]: fixed

**Review by dimaPortyanka — DISMISSED** [review]

**Review by dimaPortyanka — APPROVED** [review]

**Review by taraspaliy2020 — APPROVED** [review]

**Pull request merged**